src/main/java/wolox/training/models/Book.java

```diff
@@ -26,25 +56,52 @@
 
     @Column(nullable = false)
     private String author;
+    public void setAuthor(String author){
+        this.author = checkString(author, "author");
+    }
 
     @Column(nullable = false)
     private String image;
+    public void setImage(String image){
+        this.image = checkString(image, "image");
+    }
 
     @Column(nullable = false)
     private String title;
+    public void setTitle(String title){
+        this.title = checkString(title, "title");
+    }
 
     @Column(nullable = false)
     private String subtitle;
+    public void setSubtitle(String subtitle){
+        this.subtitle = checkString(subtitle, "subtitle");
+    }
 
     @Column(nullable = false)
     private String publisher;
+    public void setPublisher(String publisher){
+        this.publisher = checkString(publisher, "publisher");
+    }
 
     @Column(nullable = false)
     private String year;
+    public void setYear(String year){
+        this.year = checkString(year, "year");
+    }
 
     @Column(nullable = false)
     private int pages;
+    public void setPages(int pages){
+        checkArgument(
+            pages >= 0,
+            "Pages must be greater or equal than zero" );
+        this.pages = pages;
+    }
 
     @Column(nullable = false)
     private String isbn;
+    public void setIsbn(String isbn){
```

Comment:
I think this setter could be private, because it's not used from another class (This way we could make this object even more immutable 😃 )